### DISCUSSION TITLE
what is [Errno 2]

### DISCUSSION THREAD

**Discussion by tank091105** [AnthonyMichaelTDM/DRG-Save-Editor#50] [Q&A] [answered]
So I new to pip and python but I used the tutorial provided but when I got to starting it up I told me C:\Users\tank9\AppData\Local\Microsoft\WindowsApps\python.exe: can't open file 'C:\Users\tank9\src\main\python\main.py': [Errno 2] No such file or directory and I checked the first one to see if it actually worked and was greeted with this ERROR: Could not open requirements file: [Errno 2] No such file or directory: './requirements.txt' if you have any ideas what to do or how I can fix it please  thank you for reading

**Comment by AnthonyMichaelTDM** [comment, marked as answer]
You’re in the wrong directory

**Comment by meanliar** [comment]
if you don't know Python then the video in the readme is probably sufficient: https://www.youtube.com/watch?v=2h2-nZ2ptRo
